captum/attr/_utils/lrp_rules.py

```diff
@@ -33,16 +33,10 @@ def forward_hook(self, module, inputs, outputs):
     @staticmethod
     def backward_hook_activation(module, grad_input, grad_output):
         """Backward hook to propagate relevance over non-linear activations."""
-        if (
-            isinstance(grad_input, tuple)
-            and isinstance(grad_output, tuple)
-            and len(grad_input) > len(grad_output)
-        ):
-            # Adds any additional elements of grad_input if applicable
-            # This occurs when registering a backward hook on nn.Dropout
-            # modules, which has an additional element of None in
-            # grad_input
-            return grad_output + grad_input[len(grad_output) :]
```

Comment:
We don't currently, I will look into adding them in a separate diff. This shouldn't be a problem with the new approach, since that was a quirk of the module hook; we now set the hooks from the forward hook which should be fine.